### PR TITLE
Responsive support

### PR DESCRIPTION
**Pull request by thinkholic**

### CODE REVIEW

**Comment by arunoda** [comment]
Have a look at this from Chrome's responsive tools and this looks bad.

**Comment by arunoda** [comment]
If this takes time. 
Try to use existing CSS layout tool for layouting. 

It's okay to use CSS for that purpose.

**Comment by thinkholic** [comment]
Okay. noted.

**Comment by arunoda** [comment]
This seems to be a good one: flexboxgrid.com

File: src/components/Homepage/Addon/index.js [hunk 1]
```diff
@@ -2,30 +2,100 @@ import React, { Component, PropTypes } from 'react';
...
+
+  handleResize() {
+    const width = window.innerWidth;
+    const stateObj = {
+      col: false,
+      headingFontSize: 20,
+      textFontSize: 18,
+    };
+
+    if (width < 775) {
```
Comment (arunoda) [inline src/components/Homepage/Addon/index.js]: Do not do this.
Use mediaQueries with Radium.

**Comment by arunoda** [comment]
Do these things:

* Try to add some margin for the getting-started page.
* Remove the padding for addonList a bit.
* We need to set a smaller fontSize for addonList
* Make sure EXPLORE MORE and WRITE YOUR OWN links always show up in new lines.
* Rebase this with master.

**Comment by arunoda** [comment]
@thinkholic check [this machan](http://b70bf34b-4d31-415d-abdb-d800833e6b6b.go.storybooks.io/?selectedKind=Homepage&selectedStory=page&full=0&down=0&left=0&panelRight=0).

Addons alignment is kind a not exactly center. Can we do something for that?
After that we can take this in.

**Comment by thinkholic** [comment]
@arunoda noted.

**Comment by thinkholic** [comment]
@arunoda Can you check now.

**Comment by arunoda** [comment]
Cool. This is great.
Send me the docs page PR too.

**Pull request merged**